acceptance/extender_test.go

```diff
@@ -40,8 +40,12 @@ var (
 )
 
 const (
-	msgPullingRemoteImage          = "Did not find cache key, pulling remote image"
```

Comment:
I don't think this message actually ever existed in kaniko, only in my local copy 🙈